stdlib/public/runtime/Errors.cpp

```diff
@@ -149,12 +149,15 @@ void swift::dumpStackTraceEntry(unsigned index, void *framePC,
     return;
   }
 
-  // If SymbolInfo:lookup succeeded then fileName is non-null. Thus, we find the
+  // If SymbolInfo:lookup succeeded and fileName is non-null, we can find the
   // library name here. Avoid using StringRef::rsplit because its definition
   // is not provided in the header so that it requires linking with
   // libSupport.a.
-  llvm::StringRef libraryName{syminfo->getFilename()};
-  libraryName = libraryName.substr(libraryName.rfind('/')).substr(1);
+  llvm::StringRef libraryName{"<unavailable>"};
```

Comment:
This makes me slightly uncomfortable.  If this ever gets changed to a `Twine` things can go badly.

Reply:
Can you clarify what's worrying you? Did I regress something? :(

Reply:
I've refactored this not to use `llvm::StringRef` at all.